apps/schematic/api/schematic_api/controllers/storage_controller_impl.py

```diff
@@ -22,9 +22,15 @@
 from schematic_api.models.file_metadata import FileMetadata
 from schematic_api.models.file_metadata_array import FileMetadataArray
 from schematic_api.models.file_metadata_page import FileMetadataPage
-from schematic_api.controllers.utils import handle_exceptions, get_access_token
+from schematic_api.controllers.utils import (
+    handle_exceptions,
+    get_access_token,
+    purge_synapse_cache,
+)
 from schematic_api.controllers.paging import Page
 
+AWS_SYNAPSE_CACHE_PATH = "/var/tmp/synapse"
```

Comment:
Based on the code, this is not just the cache path when schematic APIs run on AWS, but this will also becomes the cache path when schematic APIs run locally? If so, I suggest changing the variable name to something else

Reply:
I beleive this only happens in the dev contianer and not anywhere else, but I cna change it just in case.